webdriver-spec.html

```diff
@@ -400,7 +400,10 @@
 
 <p>An <dfn>ASCII string</dfn> is a string
  in the range U+0000 to U+007F, inclusive.
-</section>
+
+<p>A <dfn data-lt=uuid>Universally Unique IDentifier (UUID)</dfn> is
+ a 128 bits long URN that requires no central registration process. [[RFC4122]]
```

Comment:
I wonder if we need to be need to explicit and say that we want a UUID Version 4

on page 6 of [RFC4122](https://www.ietf.org/rfc/rfc4122.txt) it says version 4 is
`The randomly or pseudo-randomly generated version specified in this document`